tests/e2e/specs/editor/media/insertMovVideo.js

```diff
@@ -26,18 +31,28 @@ describe('Inserting .mov from dialog', () => {
   it('should not list the .mov', async () => {
     await createNewStory();
     await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
+
     await expect(page).toClick('button', { text: 'Upload' });
 
     await page.waitForSelector(MODAL, {
       visible: true,
     });
-    await expect(page).toClick('button', { text: 'Media Library' });
+
+    const fileName = await uploadFile('small-video.mov', false);
+    const fileNameNoExt = fileName.replace(/\.[^/.]+$/, '');
+
     await clickButton(
       '.attachments-browser .attachments .attachment:first-of-type'
     );
 
     await expect(page).not.toMatchElement('.type-video.subtype-quicktime');
 
-    expect(page).toClick('.media-modal-close');
+    await page.keyboard.press('Escape');
+
+    await page.waitForSelector(MODAL, {
+      visible: false,
+    });
+
+    await deleteMedia(fileNameNoExt);
```

Comment:
Why doesn't `fileName` work here?

Reply:
DeleteMedia [uses the filename](https://github.com/google/web-stories-wp/pull/5849/files#diff-343f6c92f2c5e1155975691361745630b9f581d76765fb818324deb52fe69e1fR34) without the ext to delete the file. DeleteMedia was designed to be paired with [uploadMedia](https://github.com/google/web-stories-wp/pull/5849/files#diff-429e6eae0f61dc1bfcf4f19da024531f445c45c41a800fcdae5fc3a3ca221422R1) that returns [without the ext](https://github.com/google/web-stories-wp/pull/5849/files#diff-429e6eae0f61dc1bfcf4f19da024531f445c45c41a800fcdae5fc3a3ca221422R40). In this one example, it doesn't use uploadMedia, but upload file. This is because uploadFile as the mov fails to uploads ( expected of course ), we can't reuse uploadMedia.

Reply:
Ohh, got it!